src/api/spec/controllers/webui/obs_factory/staging_projects_controller_spec.rb

```diff
@@ -125,15 +138,30 @@
     end
 
     context 'with a existent factory_staging_project' do
-      subject { get :show, params: { project: factory, project_name: 'A' } }
-
       before do
         bs_request.update(state: 'declined')
         factory_staging_a.update(description: description)
       end
 
-      it { expect(subject).to have_http_status(:success) }
-      it { expect(subject).to render_template(:show) }
+      context 'requesting html' do
+        subject { get :show, params: { project: factory, project_name: 'A' } }
+
+        it { expect(subject).to have_http_status(:success) }
```

Comment:
I would use the [one-liner syntax here](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax)